dask/base.py

```diff
@@ -424,9 +426,10 @@ def _extract_graph_and_keys(vals):
     for v in vals:
         # Optimization to avoid merging dictionaries in Delayed values. Reduces
         # memory usage for large graphs.
```

Comment:
I believe this comment no longer pertains to `Delayed` values only. Should delete/update to reflect.